api/core/workflow/nodes/event/event.py

```diff
@@ -46,7 +46,7 @@ class SingleStepRetryEvent(BaseModel):
 
     inputs: dict | None = Field(..., description="input")
     error: str = Field(..., description="error")
-    outputs: dict = Field(..., description="output")
+    outputs: dict | None = Field(..., description="output")
```

Comment:
Is this `None` meaningful? Could we use a `{}` instead?

Reply:
If the passed type can be None, Pydantic will raise an error if not declared.